### PR TITLE
fixes issue #566

### PR DESCRIPTION
**Pull request by mkhahani**

### CODE REVIEW

File: erizo_controller/erizoClient/src/Connection.js [hunk 1]
```diff
@@ -115,21 +121,35 @@ Erizo.GetUserMedia = function (config, callback, error) {
...
     } else {
-      if (typeof module !== 'undefined' && module.exports) {
-        L.Logger.error('Video/audio streams not supported in erizofc yet');
-      } else {
-        navigator.getMedia(config, callback, error);
-      }
+        if (typeof module !== 'undefined' && module.exports) {
+            L.Logger.error('Video/audio streams not supported in erizofc yet');
+        } else {
+            if (config.video && Erizo.getBrowser() == 'mozilla') {
```
Comment (jcague) [inline erizo_controller/erizoClient/src/Connection.js]: jshint is complaining here: `Expected '===' and instead saw '=='.`

File: erizo_controller/erizoClient/src/Connection.js [hunk 2]
```diff
@@ -115,21 +121,35 @@ Erizo.GetUserMedia = function (config, callback, error) {
...
+                if (config.video.mandatory != undefined) {
```
Comment (jcague) [inline erizo_controller/erizoClient/src/Connection.js]: and here: `Expected '!==' and instead saw '!='.`

File: erizo_controller/erizoClient/src/Connection.js (hunk 1 above)
Comment (mkhahani) [inline erizo_controller/erizoClient/src/Connection.js]: I believe the jshint is not smart enough. Using '==' for 'mozilla' is completely safe.

Comment (jcague) [inline erizo_controller/erizoClient/src/Connection.js]: agree, just good practices (https://github.com/airbnb/javascript#comparison--eqeqeq). We now follow Airbnb style guide.

**Comment by jcague** [comment]
Nice work! thanks @mkhahani

**Pull request merged**